src/main/java/org/opensearch/security/dlic/rest/api/AuthTokenProcessorAction.java

```diff
@@ -41,10 +41,12 @@
 import java.util.Collections;
 import java.util.List;
 
+import com.google.common.collect.ImmutableList;
+
 
 public class AuthTokenProcessorAction extends AbstractApiAction {
 	private static final List<Route> routes = Collections.singletonList(
```

Comment:
`routes` should be already a modified collection, no need to change `routes()` method.

Reply:
It is the same as what we have currently in the main.

Reply:
populate `routes` with modified collection and keep `route()` method unmodified:
```
routes = addRoutesPrefix(Collections.singletonList(...));
```